docs/ansible.utils.to_paths_lookup.rst

```diff
@@ -52,7 +52,7 @@ Parameters
                     <td>
                     </td>
                 <td>
-                        <div>Prepend each path entry. Useful to add the initial <code>var</code> name.</div>
+                        <div>Prepend each path entry. Useful to add the initial <code>var</code>. name.</div>
```

Comment:
Same thing here. Or was this meant to be `var.name`? Or just `var` and delete "name"?